src/python/cli_new/lib/cli/mesos.py

```diff
@@ -169,11 +169,11 @@ def __init__(self, master, task_id):
         # "MESOS". Having a type of "MESOS" implies that it was launched by the
         # UCR -- all other types imply it was not.
         try:
-            tasks = get_tasks(master)
+            tasks = get_tasks(master, query={'task_id': task_id})
         except Exception as exception:
-            raise CLIException("Unable to get tasks from leading"
+            raise CLIException("Unable to get task {task_id} from leading"
```

Comment:
Following the documentation "task_id=VALUE        Only return tasks with this ID (should be used together with parameter 'framework_id')." in http://mesos.apache.org/documentation/latest/endpoints/master/tasks/, I would suggest returning the exception `Unable to get tasks with ID {task_id} ...`.